lib/engine/g_1860/corporation.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../corporation'
+
+module Engine
+  module G1860
```

Comment:
i'd like to avoid having a custom corporation for 1860, 

we already have the concept of receivership i think

Reply:
1860 is going to be an outlier here.
1860 corporations have corporation-specific allowed par prices, layers (order of availability to start) and at least four stage flags: Receivership, Insolvency, Bankrupt, and Closed (different from Bankrupt). The first two flags are independent of each other.

Reply:
Can you help me understand why a custom corporation for 1860 is bad?